sdk/storage/README.md

```diff
@@ -51,10 +57,20 @@ This library depends on following ES features which need external polyfills load
 - `String.prototype.includes`
 - `Array.prototype.includes`
 - `Object.keys` (Override IE11's `Object.keys` with ES6 polyfill forcely to enable [ES6 behavior](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/keys#Notes))
+- `Symbol`
+- `Symbol.asyncIterator`
+
+[polyfill.io](https://polyfill.io/v2/docs/) doesn't support `Symbol.asyncIterator` yet. One way to add it (after adding `Symbol` polyfill) is
+
+```javascript
```

Comment:
Did we add this polyfill into our code base? If so, customers may not need load this by themselves.

Reply:
Right, it will not be needed once we integrate the fix from `master` branch.

---
In reply to: [297677110](https://github.com/Azure/azure-sdk-for-js/pull/3744#discussion_r297677110) [](ancestors = 297677110)

Reply:
Removed this since we have it in product code.